apps/maintenance/pages/index.tsx

```diff
@@ -0,0 +1,53 @@
+import Image from "next/image";
+import React from "react";
+
+const MaintenancePage = () => {
+  return (
+    <div className="relative">
+      <div className="absolute inset-0 z-0">
+        <div className="absolute inset-0 bg-gradient-to-r from-purple-700 to-blue-300"></div>
```

Comment:
I think @ritacamacho should give us some ideas aswell.

Reply:
This is the current design of the page (the dojo logo rotates):
![image](https://user-images.githubusercontent.com/93675410/228532111-09e4a4c5-9ede-47b3-a945-6f0ccde2e648.png)

Reply:
> I am not a fan of the gradient. It does not match the rest of the site's visual identity. @JulioJPinto any ideas?

Yeah, I dislike it too. It would probably be better off with a flat color like purple and all the text in white.
On another note, I also dislike the animation of the logo rotating. Personally the page for me would only have an image to inform we are under maintenance. I will try to work on a SVG that looks like what I imagine. 

![image](https://user-images.githubusercontent.com/94169007/228532499-1dc2d693-a7cc-44d0-a1a6-a0069cb59778.png)

Reply:
This is currently like this @JulioJPinto, tell me how you want this. It's needed for today done this page :pray: 
![Uploading julio.png…]()

Reply:
This is currently like this @JulioJPinto, tell me how you want this. It's needed for today done this page :pray: 
![julio](https://user-images.githubusercontent.com/109802203/228816226-540342e2-26c3-45d6-a7cb-3c04269c6a20.png)

Reply:
So you want to apply flat the purple flat color and turn all the text the same white? Then instead of the logo, use a image like you suggested?

Reply:
So you want to apply flat the purple flat color and turn all the text the same white? Then instead of the logo, use a image like you suggested?